indy_common/authorize/auth_constraints.py

```diff
@@ -145,7 +153,8 @@ def from_dict(as_dict):
 
 class ConstraintCreator:
     @staticmethod
-    def create_constraint(as_dict):
+    def create_constraint(input_dict):
+        as_dict = dict(input_dict)
```

Comment:
Make deepcopy() here or before sending ```input_dict``` in ```create_constraint```

Reply:
i think, that dict() calling is enough. It's like a copy() call

Reply:
But there are 3 places where we need to do dict(). We can change it to one deepcopy().